parallel-install/pkg/deployment/deployment_test.go

```diff
@@ -240,7 +240,7 @@ func TestDeployment_StartKymaDeployment(t *testing.T) {
 func TestDeployment_StartKymaUninstallation(t *testing.T) {
```

Comment:
Wouldn't it be better if we move deletion related tests to another file, i.e. deletion_test.go?

Reply:
You got me! Was too lazy here.. will do that :)